SampleApps/MessagingDemoApp/AppDelegate.swift

```diff
@@ -42,7 +42,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UNUserNotificationCenterD
 
         let extensions = [
             Optimize.self,
-            Consent.self,
+            //Consent.self,
```

Comment:
Js confirming you want Consent extension disabled here.

Reply:
yes, it's intentional 👍